interactions.tsv

```diff
@@ -23,1329 +23,1329 @@ sourceOccurrenceId	sourceTaxonId	sourceTaxonName	sourceBodyPartId	sourceBodyPart
-	EOL:3685	Bacillariophyta 						eaten by		EOL:117	Polynoidae 											
-	EOL:3685	Bacillariophyta 						eaten by		EOL:509444	Nyctiphanes couchi 											
-	EOL:3685	Bacillariophyta 						eaten by		EOL:597887	Molpadia borealis 											
-	EOL:3685	Bacillariophyta 						eaten by		EOL:509431	Meganyctiphanes norvegica 											
-	EOL:3685	Bacillariophyta 						eaten by		EOL:456007	Lumbriclymene minor 											
-	EOL:3685	Bacillariophyta 						eaten by		EOL:14326	Erythrops 											
-	EOL:3685	Bacillariophyta 						eaten by		EOL:600551	Einhornia arctica 											
-	EOL:3685	Bacillariophyta 						eaten by		EOL:597987	Cucumaria frondosa 											
-	EOL:3685	Bacillariophyta 						eaten by		EOL:129	Cirratulidae 											
-	EOL:3685	Bacillariophyta 						eaten by		EOL:1020724	Calanus helgolandicus 											
-	EOL:3685	Bacillariophyta 						eaten by		EOL:2613063	Asellota 											
-	EOL:3685	Bacillariophyta 						eaten by		EOL:474660	Aphelochaeta marioni 											
-	EOL:3685	Bacillariophyta 						eaten by		EOL:1021376	Taphromysis bowmani 											
-	EOL:3685	Bacillariophyta 						eaten by		EOL:345055	Rhithropanopeus harrisii 											
-	EOL:3685	Bacillariophyta 						eaten by		EOL:325037	Farfantepenaeus duorarum 											
-	EOL:3685	Bacillariophyta 						eaten by		WORMS:161345	Americamysis almyra 											
-	EOL:3685	Bacillariophyta 						eaten by		EOL:42101	Labidocera 											
-	EOL:3685	Bacillariophyta 						eaten by		EOL:207803	Heterandria formosa 											
-	EOL:3685	Bacillariophyta 						eaten by		EOL:211782	Floridichthys carpio 											
-	EOL:3685	Bacillariophyta 						eaten by		EOL:14332	Cyclops 											
-	EOL:3685	Bacillariophyta 						eaten by		EOL:342513	Cyclaspis varians 											
-	EOL:3685	Bacillariophyta 						eaten by		no:match	Cricoctopus sp. 											
-	EOL:3685	Bacillariophyta 						eaten by		EOL:52177	Congeria 											
-	EOL:3685	Bacillariophyta 						eaten by		EOL:54809	Chironomus 											
-	EOL:3685	Bacillariophyta 						eaten by		EOL:451585	Brachidontes exustus 											
-	EOL:3685	Bacillariophyta 						eaten by		EOL:43170	Acartia 											
-	EOL:3685	Bacillariophyta 						eaten by		EOL:312939	Callinectes sapidus 											
-	EOL:3685	Bacillariophyta 						eaten by		EOL:1024570	Macrobrachium ohione 											
-	EOL:3685	Bacillariophyta 						eaten by		EOL:451460	Rangia cuneata 											
-	EOL:3685	Bacillariophyta 						eaten by		EOL:995122	Lagodon rhomboides 											
-	EOL:3685	Bacillariophyta 						eaten by		EOL:217614	Dorosoma petenense 											
-	EOL:3685	Bacillariophyta 						eaten by		EOL:1000279	Sardinella aurita 											
-	EOL:3685	Bacillariophyta 						eaten by		EOL:213022	Opisthonema oglinum 											
-	EOL:3685	Bacillariophyta 						eaten by		EOL:54923	Polypedilum 											
-	EOL:3685	Bacillariophyta 						eaten by		EOL:55368	Naonella 											
-	EOL:3685	Bacillariophyta 						eaten by		EOL:481634	A
```

Comment:
Rather than using http://purl.obolibrary.org/obo/RO_0002434  (interacts with) , suggest to use http://purl.obolibrary.org/obo/RO_0002437 (biotically interacts with)

Reply:
makes sense to me.
making changes to my local copy now